tests/crypto/rand32/prj_hw_random_psa_crypto.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+CONFIG_LOG=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_HARDWARE_DEVICE_CS_GENERATOR=y
+CONFIG_TFM_PARTITION_CRYPTO=y
+CONFIG_TFM_CRYPTO_RNG_MODULE_ENABLED=y
+CONFIG_ENTROPY_PSA_CRYPTO_RNG=y
```

Comment:
same fix to this file as well

Reply:
I thought about this, but then I noticed that other conf files in that folder were basically replicating all configs, so I thought there might be some (mysterious) reason for that. Btw, no problem, I'll fix it

Reply:
ok, I took a look and indeed in this case you have to leave all the configs.

The difference comes down to `OVERLAY_CONFIG` vs `CONF_FILE`. When you have `CONF_FILE` you have to specify all the options because the `prj.conf` is not considered at all. When using `OVERLAY_CONFIG` on the other hand, the overlay settings are merged with the ones from `prj.conf`.

In this case the two tests are using one `OVERLAY_CONFIG` and the other one `CONF_FILE`. No idea why. But yeah, you have to bring back these settings for the `rand32` test.

Reply:
Yes, I should have looked at the `testcase.yaml` myself to answer this, but I didn't thought about it. Sorry

Reply:
Yep it was done on purpose, no idea why those are different, this should be cleaned at some point